src/components/button/Button.styles.js

```diff
@@ -1,27 +1,22 @@
 import {StyleSheet} from 'react-native';
+import { color } from '../../theme';
 
 export const styles = StyleSheet.create({
-    buttonContainer: {
+    container: {
         borderRadius: 8,
+        alignItems: 'center',
         justifyContent: 'center',
-        width: '75%',
-        height: '5%'
     },
     buttonDisabled: {
-        backgroundColor: '#D1D1D1',
+        backgroundColor: color.lightGray
     },
     buttonEnabled: {
-        backgroundColor: '#7199FF',
+        backgroundColor: color.primary
     },
     title: {
+        color: color.white,
         fontSize: 23,
```

Comment:
nice for changing the color, try to change the sizes too.

Reply:
Would I add a default font size to the size.js theme file?